frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -263,3 +268,125 @@ it("clicking on the button only adds the email once", () => {
   expect(handleClose).toHaveBeenCalledTimes(1);
   expect(addEmail).toHaveBeenCalledTimes(1);
 });
+
+describe("edit action dialog helper file", () => {
+  // ============================== VARIABLES ==============================
+  // empty variables
+  const emptyEmailType = "";
+  const emptyEmailTo = "";
+  const emptyEmailCc = "";
+  const emptyEmailResponse = "";
+  const emptyReasonsForActionContent = "";
+  const emptyTaskContent = "";
+
+  // initial variables (for edit only)
+  const initialEmailType = EMAIL_TYPE.replyAll;
+  const initialEmailTo = "to";
+  const initialEmailCc = "cc";
+  const initialEmailResponse = "response";
+  const initialReasonsForActionContent = "reasons for action";
+  const initialTaskContent = "tasks";
+
+  // current variables
+  let emailType = EMAIL_TYPE.replyAll;
+  let emailTo = "to";
+  let emailCc = "cc";
+  let emailResponse = "response";
+  let reasonsForActionContent = "reasons for action";
+  let taskContent = "tasks";
+  // ==========================================================================
+
+  it("isEmailFormEmpty function", () => {
```

Comment:
your it statements should describe the test case you are doing, similar to how you described your test cases in comments, above. That way when we see a failure, we'll know exactly what it's about.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/229/commits/49daaf7a6c0336c363ca3bca1e25056c3930bc3e